src/test/java/com/karankumar/bookproject/utils/SecurityTestUtils.java

```diff
@@ -0,0 +1,24 @@
+package com.karankumar.bookproject.utils;
```

Comment:
Could you please add our [copyright notice](https://github.com/Project-Books/book-project/blob/master/docs/COPYRIGHT) to the top of this file?